src/main/java/service/util/ParserWriterImpl.java

```diff
@@ -0,0 +1,24 @@
+package service.util;
+
+import dao.StorageDao;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ParserWriterImpl implements ParserWriter {
```

Comment:
Are you sure you need this class? Its name doesn't correspond to its function - replacing some symbols in input